frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -75,6 +79,44 @@ const getInventoryData = resources => {
   };
 };
 
+const getInfrastructureData = (infrastructure) => {
```

Comment:
we send other resources to the ClusterOverview and select all the data we need in there. Can't we do the same for InfrastructureModel and move this logic to web-ui-components?

Reply:
web-ui-component `Details` is rendering a description list and it is expecting simple key-value pairs. I do not want to put specific login for `Infrastructure Model` into `Details` as it would make it less generic and reusability for other overviews would suffer. I could create `ClusterDetails` in web-ui-components, which would contain logic from `getInfrastructureData` function and pass the extracted data to `Details`. WDYT ?

Reply:
IMO having a clear separation of concerns is a better way. 

reusability issue: we can wrap the functionality inside the components and others just pick the right layer

Reply:
logic moved to ClusterDetails in web-ui-components